### ISSUE TITLE
Remove back arrow button on first page

### ISSUE THREAD

**Issue by kuzunguka** [mimbrown/render#97]
The maroon "back" arrow button on the first page of the app (Aghaz-e-Kalam article) naturally does not function. Can we remove it?

**Comment by kuzunguka** [comment]
Similarly, the last page of the app (maps) has a non-functioning "forward" arrow button. Can we remove it?

**Comment by mnjames** [comment]
I agree with the suggestion, though I would place it as "low priority" unless it's easy to do.
